x/mongo/driver/topology/topology_errors_test.go

```diff
@@ -4,6 +4,7 @@
 // not use this file except in compliance with the License. You may obtain
 // a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
 
+//go:build go1.13
```

Comment:
Is this necessary?

Reply:
It's not, but start with go 1.17 this is suggested way to add build tags. So I think there should be a period when `//go:build` and `// +build` are used together and then `// +build` is removed. In my case `//go:build` was automatically added on save by vsc extension that I disable to remove this line.